modules/common.nix

```
@@ -195,15 +207,23 @@ in {
         (pkgs.writeShellScript "load-env")
       ];
 
-    assertions = [
-      {
-        assertion = cfg.user != "";
-        message = "A user must be configured in 'hjem.users.<user>.name'";
-      }
-      {
-        assertion = cfg.directory != "";
-        message = "A home directory must be configured in 'hjem.users.<user>.directory'";
-      }
-    ];
+    assertions =
+      [
+        {
+          assertion = cfg.user != "";
+          message = "A user must be configured in 'hjem.users.<user>.name'";
+        }
+        {
+          assertion = cfg.directory != "";
+          message = "A home directory must be configured in 'hjem.users.<user>.directory'";
+        }
+      ]
+      ++ (mapAttrsToList (
+          _target: file: {
```

```suggestion
          _: file: {
```
If you're not using a variable, I don't see the point of keeping its name.

---

Some people prefer to keep the name for legibility, have no strong opinion on it. Will update it once I know whether we go forward with this PR

---

Sounds good.

---

I have no opinion, either is fine, the underscore-prefix is a soft indicator that it's unused (and a convention that is shared in nixpkgs' lib).